tests/runfiles/common.run

```diff
@@ -642,6 +642,10 @@ tags = ['functional', 'compression']
 tests = ['cp_files_001_pos', 'cp_files_002_pos', 'cp_stress']
 tags = ['functional', 'cp_files']
 
+[tests/functional/zap_shrink]
+tests = ['zap_shrink_001_pos']
+tags = ['functional', 'zap_shrink']
```

Comment:
You will need to add the new scripts to `tests/zfs-tests/tests/Makefile.am` so they're included in the tarball and can by run by the CI.

Reply:
done